src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
+    private static final Random random = new Random(47);
```

Comment:
Avoid using static fields for mutable objects. In this case, 'random' should be an instance field, not a static one, as per the checklist guideline to not use static methods or fields unnecessarily.